### ISSUE TITLE
bash crashes in LibC on ctrl-d

### ISSUE THREAD

**Issue by nico** [SerenityOS/serenity#2569]
1. Type something (`asdf`)
2. Hit ctrl-a to go to start of line
3. hit ctrl-d to delete a character

Actual:
```
bash-5.0$ ASSERTION FAILED: false
../Libraries/LibC/termcap.cpp:144
```

**Comment by Nicholas-Baron** [comment]
That line is an [`ASSERT_NOT_REACHED`](https://github.com/SerenityOS/serenity/blob/10fd862a5562ff96d8abd45c46b9ff17e24820a5/Libraries/LibC/termcap.cpp#L139)(probably with TODO semantics)

It seems that bash calls `tgoto`, which seems to be a function for formatting in the terminal.

**Comment by IdanHo** [comment]
`tgoto` was implemented in c762a0c4d68bfc7de9dc4dea9afdad6870bd4d58 and bash does not crash anymore
